App/Containers/ThreadsList.js

```diff
@@ -109,18 +117,13 @@ class ThreadsList extends React.PureComponent {
         )}
         {this.props.threads.length !== 0 && (
           // FIXME: This should be a FlatList for sure
-          <ScrollView
-            style={styles.contentContainer}
-            refreshControl={
-              <RefreshControl
```

Comment:
Not sure if you wanted to drop refresh here, but FlatList supports it if you want it.

Reply:
ah for sure, will add back